cf_remote/commands.py

```diff
@@ -931,3 +932,14 @@ def agent(hosts, bootstrap=None):
             print(output)
 
     return 0
+
+
+def connect_cmd(hosts):
+
+    if len(hosts) > 1:
+        user_error("You can only connect to one host at a time")
```

Comment:
@victormlg what happens if the user doesn't specify any hosts?

Reply:
The parser in the main function should handle this no ? When I try on my pc I get 
```
python3 cf_remote connect 
usage: cf_remote connect [-h] --hosts HOSTS
cf_remote connect: error: the following arguments are required: --hosts/-H
```

Reply:
Okay, that's good, but unclear rom reading your code, when I read the function, it looks like those other cases are not handled. I would add some comments / assertions;

```suggestion
    assert hosts and len(hosts) >= 1 # Ensured by argument parser
    if len(hosts) > 1:
        user_error("You can only connect to one host at a time")
```